docs/analyticslib.md

```diff
@@ -0,0 +1,488 @@
+The following documentation describes the use of the forward fills, pivot, racking, and intervals
```

Comment:
Document needs a header, and also added to the mkdocs.yml to come up as a separate page. 

We need a bit more background - this is going to be an analytics lib which we will augment, this is the start point.